Forwarded-Upstream: http://code.djangoproject.com/changeset/11619
Comment:
 Fix FTBFS under Python 2.6.3; int(Decimal('nan')) raises ValueError instead
 of returning None.

--- ./django/template/defaultfilters.py (revision 10543)
+++ ./django/template/defaultfilters.py (revision 11619)
@@ -163,5 +163,5 @@
     try:
         m = int(d) - d
-    except (OverflowError, InvalidOperation):
+    except (ValueError, OverflowError, InvalidOperation):
         return input_val
 
